pyvalidator/is_mailto.py

```diff
@@ -0,0 +1,18 @@
+import re
+
+def is_valid_mailto_link(link):
```

Comment:
Do define the expected types of the input argument and the output.

In addition, since the input is a string (url), we should utilise the:
assert_string method, which makes sure that the input argument is of the given type and also does cast it to a custom String type, which does inherit from the original string type.